### ISSUE TITLE
Add region label to prefooter region(s)

### ISSUE THREAD

**Issue by joewhitsitt** [uiowa/uiowa#6297]
# Story

<img width="966" alt="Screenshot 2023-03-24 at 1 02 34 PM" src="https://user-images.githubusercontent.com/4663676/227605201-456d2243-1f7f-4afe-9f26-47dec11c3584.png">

As a user relying on assistive technologies, I can get descriptive information about the region I just navigated to.

# Proposed solution

- [ ] For this role region, add a label based on the content within (tags, related by tag, prefooter content, etc.)

**Comment by bspeare** [comment]
Dentistry emailed its-web about this issue coming up in their reports using the related content feature on the articles content type.   Siteimprove recommends `<div role="region" aria-label="Related content"></div>`

**Comment by bspeare** [comment]
Yes, I think if we switched these to `<aside>` or `role="complementary"` the error would go away.

**Comment by briand44** [comment]
Registrar reported this issue as well.

Page region missing an accessible name: [https://my2.siteimprove.com/Accessibility/995867/NextGen/Issue/1?ruleId=40&pageSegments=&issueKind=1&exceptTags=1,2&conformance=&siteTargetIssueKinds=1,2&wcagVersion=21&siteTarget.wcagVersion=21&siteTarget.conformanceLevels=0,1,3,4&siteTarget.issueKinds=1,2](https://nam12.safelinks.protection.outlook.com/?url=https%3A%2F%2Fmy2.siteimprove.com%2FAccessibility%2F995867%2FNextGen%2FIssue%2F1%3FruleId%3D40%26pageSegments%3D%26issueKind%3D1%26exceptTags%3D1%2C2%26conformance%3D%26siteTargetIssueKinds%3D1%2C2%26wcagVersion%3D21%26siteTarget.wcagVersion%3D21%26siteTarget.conformanceLevels%3D0%2C1%2C3%2C4%26siteTarget.issueKinds%3D1%2C2&data=05%7C02%7Cbrian-dickey%40uiowa.edu%7Ca13bda7b6932452f63ef08ddc56afbe7%7C1bc445959aba4fc3b8ec7b94a5586fdc%7C1%7C0%7C638883785462368204%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C0%7C%7C%7C&sdata=5vu56K75TKifiOl1YC2aJG1p95Vt3vTP8Q5MePwM7WQ%3D&reserved=0)
